Applied changes to wire publisher.

MMaiero opened the pull request:
Fixed the topic in comp. properties.
Defined code to perform the string substitution when the assetName property
is provided.
Modified the WireAsset in order to provide only one entry in WireEnvelope where
the asset name is specified.

Signed-off-by: MMaiero <matteo.maiero@eurotech.com>

MMaiero commented:
Closes #1246

amitjoy commented:
will review it as soon as I finish reviewing PR #1292

dwoodard1 commented:
@amitjoy That was a mistake, didn't mean to click your name. I will take care of this review.

amitjoy commented:
@dwoodard1 👍

dwoodard1 commented:
@MMaiero @amitjoy The PR looks good, but I wanted to try a few scenarios. However, the modbus driver from the Eclipse Marketplace no longer seems compatible. Should I be using a newer version of the driver?

amitjoy commented:
You can use the OPC-UA driver from the current develop branch.

amitjoy commented:
Due to the changes in PR #1296, I am certain OPC-UA driver would not work as well. The necessary changes for the OPC-UA Driver to work are in #1292.

dwoodard1 commented:
Yes, it is also broken. I will approve the review the review and wait to merge until the other PRs have cleared.

MMaiero committed:
Applied changes to wire publisher.

Fixed the topic in comp. properties.
Defined code to perform the string substitution when the assetName property
is provided.
Modified the WireAsset in order to provide only one entry in WireEnvelope where
the asset name is specified.

Signed-off-by: MMaiero <matteo.maiero@eurotech.com>
